packages/block-library/src/post-author/index.php

```diff
@@ -24,7 +24,9 @@ function post_author_build_css_colors( $attributes ) {
 
```

Comment:
This PR is also a good opportunity to implement this https://github.com/WordPress/gutenberg/pull/22872#discussion_r437930625

The idea being that you don't need to touch any backend or frontend code when applying the "useColors" flag, everything should just work.

Reply:
Since the classes/styles for this one have already been manually set-up, I wouldn't be opposed to merging this and circling back to update it after we have enabled the proper support and updated the other FSE blocks on the task list.

Reply:
For reference, started work on the render function here https://github.com/WordPress/gutenberg/pull/23007/files#diff-1334f2cb73e38f2f89d2aaf763f07ed9R307